### ISSUE TITLE
Not able to use arrows to navigate hotgraphic pins

### ISSUE THREAD

**Issue by joe-allen-89** [adaptlearning/adapt-contrib-hotgraphic#257]
### Subject of the issue/enhancement/features
When navigating the hotgraphic using a keyboard you're unable to use the arrows to navigate as you can with the hotgrid. Also when closing a popup the screen reader loses focus and the user is still not able to navigate with arrows. 

Seems to be linked to aria-haspopup.

### Your environment
* v5.24.8
* chrome + Jaws 2023

### Steps to reproduce
Navigate to hotgraphic with screen reader, try and move between pins with arrows, open 

### Expected behaviour
Should navigate between pins when using arrows. 

### Actual behaviour
Page scrolls.

**Comment by StuartNicholls** [comment]
@joe-allen-89 Is this due to Jaws going in and out of forms mode on HTML button elements with aria-hadpopup=dialog?

Possible solution here? I says Firefox, but not clear if it works in all browsers with Jaws?

https://github.com/FreedomScientific/VFO-standards-support/issues/82

**Comment by oliverfoster** [comment]
Please remove any aria that isn't cross supported and make a note of it.

**Comment by StuartNicholls** [comment]
https://github.com/w3c/aria-practices/issues/1383
